Hestia/app/src/main/java/hestia/backend/Cache.java

```diff
@@ -0,0 +1,95 @@
+package hestia.backend;
+
+import java.util.ArrayList;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class Cache {
+    /**
```

Comment:
We should add Javadoc to this to explain that it contains the devices and methods to access them.